refract-spec.md

```diff
@@ -534,7 +518,7 @@ Additionally, the `select` element is provided to give multiple possibilities fo
 
 The `extend` element provides a way to a way to multiply inherit one or more elements to form a new element. The `extend` element MUST NOT affect the original elements being extended, and MUST define a new instance of an element.
 
-The `extend` element MUST do a deep merge of the elements found in the `content`, but MUST NOT include the `id`, `namespaces` and `prefix` meta attributes from the content elements in the final element. Each element found in the `content` MUST derive from the same primitive element. The elements are merged from first to last.
+The `extend` element MUST do a deep merge of the elements found in the `content`, but MUST NOT include the `id` meta attribute from the content elements in the final element. Each element found in the `content` MUST derive from the same primitive element. The elements are merged from first to last.
```

Comment:
I think they are two different questions.

1. How do you handle semantic collisions? 
1. When you extend elements, is the only thing now that doesn't get included in the final element the `id`?

To the first question, I think it's defined by the user. In an HTTP API, this should be resolved in the definition of the media type, the profile, or by linking to a profile. For instance, you could do this.

```json
{
  "element": "array",
  "content": [
    {
      "element": "foo",
      "meta": {
        "link": [
          {
            "element": "link",
            "attributes": {
              "relation": "profile",
              "href": "http://example.com/profile/foo1"
            }
          }
        ]
      }
    },
    {
      "element": "foo",
      "meta": {
        "link": [
          {
            "element": "link",
            "attributes": {
              "relation": "profile",
              "href": "http://example.com/profile/foo2"
            }
          }
        ]
      }
    }
  ]
}
```